src/Microsoft.OData.Core/Metadata/EdmLibraryExtensions.cs

```diff
@@ -1868,6 +1986,19 @@ private static int InheritanceLevelFromSpecifiedInheritedType(this IEdmStructure
 
             return inheritanceLevelsFromBase;
         }
+
+        private static void EnsureOperationBoundWithBindingParameter(this IEdmOperation operation)
+        {
+            if (!operation.IsBound)
+            {
+                throw new ODataException(ErrorStrings.EdmLibraryExtensions_UnBoundOperationsFoundFromIEdmModelFindMethodIsInvalid(operation.Name));
+            }
+
+            if (operation.Parameters.FirstOrDefault() == null)

```

Comment:
>.FirstOrDefault() == null [](start = 36, length = 25)

Is FirstOrDefault() more efficient than Any()?  You don't actually need the parameter, you just need to know if one exists, right?

Reply:
They are the same. They just differ in their return types. Any is better suited though. 

---
In reply to: [370725688](https://github.com/OData/odata.net/pull/1602#discussion_r370725688) [](ancestors = 370725688)